Xplat/src/main/resources/assets/botania/lang/de_de.json

```diff
@@ -5,52 +5,630 @@
+    "botaniamisc.scaleChange": "Skalierung ändern",
+    "botaniamisc.invalidDodge": "Ungültiges Täuschungsmaneuver-Paket",
+    "botaniamisc.rannuncarpus.state_sensitive": "Exakten Blockzustand vergleichen",
+    "botaniamisc.rannuncarpus.state_insensitive": "Nur Block vergleichen",
+    "botaniamisc.lokiRingLimitReached": "Auswahlbegrenzung erreicht",
+    "botaniamisc.loonium.not_attuned": "Nicht abgestimmt",
+    "botaniamisc.loonium.attuned": "Abgestimmt",
+    "botaniamisc.loonium.attuned_one": "Abgestimmt auf Bauwerk: %s",
+    "botaniamisc.loonium.attuned_many": "Abgestimmt auf mehrere Bauwerke",
+    "botaniamisc.pollidisiac.feed_adults": "Füttere erwachsene Tiere",
+    "botaniamisc.pollidisiac.feed_babies": "Füttere Tierkinder",
+    "botaniamisc.pollidisiac.feed_all": "Füttere alle Tiere",
+
+    "botania.tater_birthday.0": "Wow, ist das für mich?",
+    "botania.tater_birthday.1": "Heute ist mein Geburtstag, ich bin jetzt %d Jahre alt!",
+    "botania.tater_birthday.2": "Vielen Dank! Ich hoffe das war auch für dich ein gutes Jahr.",
+    "botania.tater_birthday.3": "Denk immer daran: Ich glaube an dich, und du kannst es schaffen! \u2764",
+    "botania.tater_birthday.4": "Juhu!!!",
+
+
+
+    
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+    "botania.nei.brewery": "Botanische Brauerei",
+    "botania.nei.elvenTrade": "Elfenhandel",
+    "botania.nei.floatingFlowers": "Schwebende Blumen",
+    "botania.nei.manaPool": "Mana-Infusion",
+    "botania.nei.petalApothecary": "Blütenapotheke",
+    "botania.nei.runicAltar": "Runenaltar",
+    "botania.nei.pureDaisy": "Reines Gänseblümchen",
+    "botania.nei.lexica": "Lexica Botania",
+    "botania.nei.orechid": "Erchidee",
+    "botania.nei.orechidIgnem": "Flammende Erchidee",
+    "botania.nei.marimorphosis": "Tagemorphes",
+    "botania.nei.terraPlate": "Terrestrische Agglomeration",
+
+    "emi.category.botania.petal_apothecary": "Blütenapotheke",
+    "emi.category.botania.mana_infusion": "Mana-Infusion",
+    "emi.category.botania.runic_altar": "Runenaltar",
+    "emi.category.botania.terrestrial_agglomeration": "Terrestrische Agglomeration",
+    "emi.category.botania.elven_trade": "Elfenhandel",
+    "emi.category.botania.botanical_brewery": "Botanische Brauerei",
+    "emi.category.botania.pure_daisy": "Reines Gänseblümchen",
+    "emi.category.botania.orechid": "Erchidee",
+    "emi.category.botania.orechid_ignem": "Flammende Erchidee",
+    "emi.category.botania.marimorphosis": "Tagemorphes",
+
+    "tag.botania.petals.white": "Weiße Blütenblätter",
+    "tag.botania.petals.orange": "Orange Blütenblätter",
+    "tag.botania.petals.magenta": "Magentafarbene Blütenblätter",
+    "tag.botania.petals.light_blue": "Hellblaue Blütenblätter",
+    "tag.botania.petals.yellow": "Gelbe Blütenblätter",
+    "tag.botania.petals.lime": "Hellgrüne Blütenblätter",
+    "tag.botania.petals.pink": "Pinke Blütenblätter",
```

Comment:
minor, but "pink" things usually get translated into "rosa" in minecraft, as seen in the vanilla translations as well (don't ask me why)